### DISCUSSION TITLE
cant get macbook 6,2 into safe mode <title>

### DISCUSSION THREAD

**Discussion by moranjaccov** [community/community#47083] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

i have a macbook pro 6,2 mid 2010, i had catalina with patch updater and all was working fine, updated to ventura 
and then my mac started to crash black screen and the keyboard lights on... GPU panic... made a clean erase and did all once again to see if i should downgrade, went back to montiray same crash, went back to big sur, same, then i said maybe it cant handle the open core patcher i should go back to the catalina patcher, then it started to do the same with catalina... i wanted to try the safe mode and instal the patches, but it wont  load, loading for ever wont get to the login screen, ivoult off, software password off, dont know what to do it keeps on freezing, please help

panic(cpu 2 caller 0xffffff7f856cb83d): "GPU Panic: mux-regs 5 3 7f 0 0 0 severity 3 WS-ready 1 switch-state 0 IG FBs 0 EG FBs 1:3 power-state 0 3D idle HDA idle system-state 1 power-level 20:20 connect-change 0 : NVRM[0/1:0:0]: Read Error 0x00000100: CFG 0xffffffff 0xffffffff 0xffffffff, BAR0 0xd2000000 0xffffff9120557000 0x0a5480a2, D0, P3/4\n"@/BuildRoot/Library/Caches/com.apple.xbs/Sources/AppleGraphicsControl/AppleGraphicsControl-3.28.4/src/AppleMuxControl/kext/GPUPanic.cpp:170
Backtrace (CPU 2), Frame : Return Address
0xffffff91202033f0 : 0xffffff800311868d mach_kernel : _handle_debugger_trap + 0x49d
0xffffff9120203440 : 0xffffff8003252ab5 mach_kernel : _kdp_i386_trap + 0x155
0xffffff9120203480 : 0xffffff800324463e mach_kernel : _kernel_trap + 0x4ee
0xffffff91202034d0 : 0xffffff80030bea40 mach_kernel : _return_from_trap + 0xe0
0xffffff91202034f0 : 0xffffff8003117d57 mach_kernel : _DebuggerTrapWithState + 0x17
0xffffff91202035f0 : 0xffffff8003118147 mach_kernel : _panic_trap_to_debugger + 0x227
0xffffff9120203640 : 0xffffff80038bf2bc mach_kernel : _panic + 0x54
0xffffff91202036b0 : 0xffffff7f856cb83d com.apple.driver.AppleMuxControl + 59453
0xffffff9120203710 : 0xffffff7f865a6441 com.apple.nvidia.classic.NVDAResmanTesla : _osWriteRegistryDword + 0x321
0xffffff91202037d0 : 0xffffff7f866721ca com.apple.nvidia.classic.NVDAResmanTesla : _gpuExecRegOps + 0xeefd
0xffffff9120203810 : 0xffffff7f86672235 com.apple.nvidia.classic.NVDAResmanTesla : _gpuExecRegOps + 0xef68
0xffffff9120203890 : 0xffffff7f868e27f0 com.apple.nvidia.classic.NVDANV50HalTesla : __ZN11NVDANV50HAL5probeEP9IOServicePi + 0x123d22
0xffffff91202039b0 : 0xffffff7f86693d35 com.apple.nvidia.classic.NVDAResmanTesla : _constructObjEHeap + 0x51d3
0xffffff91202039d0 : 0xffffff7f865ad048 com.apple.nvidia.classic.NVDAResmanTesla : _NvRmDupObject + 0x3b5
0xffffff9120203a80 : 0xffffff7f865aaa84 com.apple.nvidia.classic.NVDAResmanTesla : _insert_registration_func + 0x240
0xffffff9120203c70 : 0xffffff7f865abc41 com.apple.nvidia.classic.NVDAResmanTesla : _NvRmControl + 0x84
0xffffff9120203d50 : 0xffffff7f8655761e com.apple.nvidia.classic.NVDAResmanTesla : __ZN9NVDATesla12GetGpuSensorEPj + 0x5e
0xffffff9120203db0 : 0xffffff7f86b69545 com.apple.driver.AGPM : __ZN4AGPM12timerHandlerEv + 0x665
0xffffff9120203e00 : 0xffffff7f86b698d6 com.apple.driver.AGPM : __ZN4AGPM23timerEventSourceHandlerEv + 0xe
0xffffff9120203e20 : 0xffffff800382fb29 mach_kernel : __ZN18IOTimerEventSource15timeoutSignaledEPvS0_ + 0x89
0xffffff9120203e90 : 0xffffff800382fa49 mach_kernel : __ZN18IOTimerEventSource17timeoutAndReleaseEPvS0_ + 0x99
0xffffff9120203ec0 : 0xffffff800315a645 mach_kernel : _thread_call_delayed_timer + 0xec5
0xffffff9120203f40 : 0xffffff800315a171 mach_kernel : _thread_call_delayed_timer + 0x9f1
0xffffff9120203fa0 : 0xffffff80030be13e mach_kernel : _call_continuation + 0x2e
      Kernel Extensions in backtrace:
         com.apple.driver.AppleMuxControl(3.28.4)[EB8D7A77-E073-36EB-A5B1-A3672CA840A3]@0xffffff7f856bd000->0xffffff7f856d0fff
            dependency: com.apple.driver.AppleGraphicsControl(3.28.4)[200F1816-A734-3977-B7AE-7A6B2363059F]@0xffffff7f85668000
            dependency: com.apple.iokit.IOACPIFamily(1.4)[2956198D-24F2-3790-A9B2-1EAB9434B906]@0xffffff7f83cf0000
            dependency: com.apple.iokit.IOPCIFamily(2.9)[2F37AE58-E6B9-3B18-9092-3B80D34C334B]@0xffffff7f83d02000
            dependency: com.apple.iokit.IOGraphicsFamily(519.15)[D5F2A20D-CAB0-33B2-91B9-E8755DFC34CB]@0xffffff7f8566b000
            dependency: com.apple.AppleGraphicsDeviceControl(3.28.4)[400C2E21-0880-36D7-AC5E-6B9BC885F158]@0xffffff7f856b4000
         com.apple.nvidia.classic.NVDAResmanTesla(10.0.4)[F8258F0F-61AD-30F4-BDD5-B18CCAAE535A]@0xffffff7f86555000->0xffffff7f867bdfff
            dependency: com.apple.iokit.IOPCIFamily(2.9)[2F37AE58-E6B9-3B18-9092-3B80D34C334B]@0xffffff7f83d02000
            dependency: com.apple.iokit.IONDRVSupport(519.15)[B419F958-11B8-3F7D-A31B-A72166B6E234]@0xffffff7f85fca000
            dependency: com.apple.iokit.IOGraphicsFamily(519.15)[D5F2A20D-CAB0-33B2-91B9-E8755DFC34CB]@0xffffff7f8566b000
         com.apple.nvidia.classic.NVDANV50HalTesla(10.0.4)[5DCF031B-F81F-32A3-ADCD-18E16EAC7CF0]@0xffffff7f867be000->0xffffff7f86a66fff
            dependency: com.apple.nvidia.classic.NVDAResmanTesla(10.0.4)[F8258F0F-61AD-30F4-BDD5-B18CCAAE535A]@0xffffff7f86555000
            dependency: com.apple.iokit.IOPCIFamily(2.9)[2F37AE58-E6B9-3B18-9092-3B80D34C334B]@0xffffff7f83d02000
         com.apple.driver.AGPM(110.23.46)[9A7BD08F-08E4-3DB7-A339-706609B0149A]@0xffffff7f86b65000->0xffffff7f86b83fff
            dependency: com.apple.iokit.IOPCIFamily(2.9)[2F37AE58-E6B9-3B18-9092-3B80D34C334B]@0xffffff7f83d02000
            dependency: com.apple.driver.IOPlatformPluginFamily(6.0.0d8)[5285E17D-5C27-38C4-B7F9-44E28D09C609]@0xffffff7f8571c000
            dependency: com.apple.iokit.IONDRVSupport(519.15)[B419F958-11B8-3F7D-A31B-A72166B6E234]@0xffffff7f85fca000
            dependency: com.apple.iokit.IOGraphicsFamily(519.15)[D5F2A20D-CAB0-33B2-91B9-E8755DFC34CB]@0xffffff7f8566b000
            dependency: com.apple.AppleGraphicsDeviceControl(3.28.4)[400C2E21-0880-36D7-AC5E-6B9BC885F158]@0xffffff7f856b4000

BSD process name corresponding to current thread: kernel_task
Boot args: keepsyms=1 debug=0x100 -lilubetaall -btlfxallowanyaddr ipc_control_port_options=0 -nokcmismatchpanic amfi=0x80 -no_compat_check amfi_allow_any_signature=1

Mac OS version:
19H15

Kernel version:
Darwin Kernel Version 19.6.0: Thu Oct 29 22:56:45 PDT 2020; root:xnu-6153.141.2.2~1/RELEASE_X86_64
Kernel UUID: 9B5A7191-5B84-3990-8710-D9BD9273A8E5
Kernel slide:     0x0000000002e00000
Kernel text base: 0xffffff8003000000
__HIB  text base: 0xffffff8002f00000
System model name: MacBookPro6,2 (Mac-F22586C8)
System shutdown begun: NO
Panic diags file available: YES (0x0)

System uptime in nanoseconds: 4129506473403
last loaded kext at 2002921153847: @filesystems.msdosfs	1.10 (addr 0xffffff7f859b4000, size 69632)
last unloaded kext at 2261238695145: >!UAudio	323.4 (addr 0xffffff7f85924000, size 434176)
loaded kexts:
>!U!SCoexistentDriver	489.120.1
>AudioAUUC	1.70
@fileutil	20.036.15
@filesystems.autofs	3.0
@AGDCPluginDisplayMetrics	3.28.4
>!AHV	1
|IOUserEthernet	1.0.1
|IO!BSerialManager	7.0.6f7
>pmtelemetry	1
>!AUpstreamUserClient	3.6.8
>!AMikeyHIDDriver	131
>AGPM	110.23.46
@Dont_Steal_Mac_OS_X	7.0.0
@GeForceTesla	10.0.4
>!A!IHDGraphics	10.0.4
>AGDCBacklightControl	5.2.6
>!AMikeyDriver	281.52
>!AHDA	281.52
>eficheck	1
>AirPort.Brcm4331	800.21.31
>!ASMCPDRC	1.0.0
>SMCMotionSensor	3.0.4d1
|!ABCM5701Ethernet	10.3.5
>!ASMCLMU	212
>!ALPC	3.1
>!AFWOHCI	5.6.2
>ACPI_SMC_PlatformPlugin	1.0.0
>!A!IHDGraphicsFB	10.0.4
>!A!ISlowAdaptiveClocking	4.0.0
>!AMCCSControl	1.5.6
>!AMuxControl	3.28.4
>!UCardReader	489.120.1
>!UTCButtons	255
>!UTCKeyboard	255
>!AIR!C	360
@filesystems.apfs	1412.141.1
>!AVirtIO	1.0
@filesystems.hfs.kext	522.100.5
@!AFSCompression.!AFSCompressionTypeDataless	1.0.0d1
@BootCache	40
@!AFSCompression.!AFSCompressionTypeZlib	1.0.0
|SCSITaskUserClient	422.120.3
@private.KextAudit	1.0
>!AAHCIPort	341.140.1
>!ASmartBatteryManager	161.0.0
>!ARTC	2.0
>!AACPIButtons	6.1
>!AHPET	1.8
>!ASMBIOS	2.1
>!AACPIEC	6.1
>!AAPIC	1.7
>!A!ICPUPowerManagementClient	222.0.0
$!AImage4	1
@nke.applicationfirewall	303
$TMSafetyNet	8
@!ASystemPolicy	2.0.0
>!A!ICPUPowerManagement	222.0.0
|EndpointSecurity	1
|IOUSBMass!SClass	4.0.4
@kext.triggers	1.0
|IOAVB!F	850.1
|IO!B!F	7.0.6f7
|IO!BPacketLogger	7.0.6f7
@plugin.IOgPTPPlugin	840.3
|IOSkywalk!F	1
@nvidia.classic.NVDANV50HalTesla	10.0.4
@nvidia.classic.NVDAResmanTesla	10.0.4
>DspFuncLib	281.52
@kext.OSvKernDSPLib	529
|IOFireWireIP	2.3.0
@!AGPUWrangler	3.28.4
>!ABacklightExpert	1.1.0
|IONDRVSupport	519.15
>!ASMBusPCI	1.0.14d1
>!AHDA!C	281.52
|IOHDA!F	281.52
|IOEthernetAVB!C	1.1.0
|IO80211!F	1200.12.2
>mDNSOffloadUserClient	1.0.1b8
>corecapture	1.0.4
|IOFireWire!F	4.7.5
>IOPlatformPluginLegacy	1.0.0
>IOPlatformPlugin!F	6.0.0d8
|IOSlowAdaptiveClocking!F	1.0.0
>!ASMBus!C	1.0.18d1
@!AGraphicsDeviceControl	3.28.4
|IOGraphics!F	519.15
>!AGraphicsControl	3.28.4
>!UMultitouch	264
>usb.IOUSBHostHIDDevice	1.2
>usb.networking	5.0.0
>usb.!UHostCompositeDevice	1.2
>usb.!UHub	1.2
|IOAudio!F	300.2
@vecLib.kext	1.2.0
|IOSerial!F	11
|IOSurface	269.11
@filesystems.hfs.encodings.kext	1
|IOSCSIMultimediaCommandsDevice	422.120.3
|IOBD!S!F	1.8
|IODVD!S!F	1.8
|IOCD!S!F	1.8
|IOAHCISerialATAPI	268
|IOAHCIBlock!S	316.100.5
>usb.!UHostPacketFilter	1.0
|IOUSB!F	900.4.2
|IOAHCI!F	290.0.1
>usb.!UEHCIPCI	1.2
>usb.!UEHCI	1.2
>!AEFINVRAM	2.1
>!AEFIRuntime	2.1
|IOSMBus!F	1.1
|IOHID!F	2.0.0
$quarantine	4
$sandbox	300.0
@kext.!AMatch	1.0.0d1
>DiskImages	493.0.0
>!AFDEKeyStore	28.30
>!AEffaceable!S	1.0
>!ASSE	1.0
>!AKeyStore	2
>

**Comment by moranjaccov** [comment]
and again, i cant live like this :( 

panic(cpu 1 caller 0xffffff7f88eedfb7): GPU Panic: mux-regs 5 3 7f 0 0 0 severity 3 WS-ready 1 switch-state 0 IG FBs 0 EG FBs 1:3 power-state 0 3D idle HDA idle system-state 1 power-level 20:20 power-retry 0 connect-change 0 : NVRM[0/1:0:0]: Read Error 0x00000100: CFG 0xffffffff 0xffffffff 0xffffffff, BAR0 0xd2000000 0xffffffb730645000 0x0a5480a2, D0, P3/4

Panicked task 0xffffff8bac331030: 9 threads: pid 131: WindowServer
Backtrace (CPU 1), panicked thread: 0xffffff9079489540, Frame : Return Address
0xfffffff2aaadadc0 : 0xffffff8007e7bebd mach_kernel : _handle_debugger_trap + 0x41d
0xfffffff2aaadae10 : 0xffffff8007fde5e6 mach_kernel : _kdp_i386_trap + 0x116
0xfffffff2aaadae50 : 0xffffff8007fcd953 mach_kernel : _kernel_trap + 0x4d3
0xfffffff2aaadaea0 : 0xffffff8007e1ba70 mach_kernel : _return_from_trap + 0xe0
0xfffffff2aaadaec0 : 0xffffff8007e7c28d mach_kernel : _DebuggerTrapWithState + 0xad
0xfffffff2aaadafe0 : 0xffffff8007e7ba46 mach_kernel : _panic_trap_to_debugger + 0x2b6
0xfffffff2aaadb040 : 0xffffff80087148b3 mach_kernel : _panic + 0x84
0xfffffff2aaadb130 : 0xffffff7f88eedfb7 com.apple.driver.AppleMuxControl : _Debugger + 0x6dcfb7
0xfffffff2aaadb210 : 0xffffff7f89ab4441 com.apple.nvidia.classic.NVDAResmanTesla : _osWriteRegistryDword + 0x321
0xfffffff2aaadb2d0 : 0xffffff7f89b801ca com.apple.nvidia.classic.NVDAResmanTesla : _gpuExecRegOps + 0xeefd
0xfffffff2aaadb310 : 0xffffff7f89b80235 com.apple.nvidia.classic.NVDAResmanTesla : _gpuExecRegOps + 0xef68
0xfffffff2aaadb390 : 0xffffff7f898df7f0 com.apple.nvidia.classic.NVDANV50HalTesla : __ZN11NVDANV50HAL5probeEP9IOServicePi + 0x123d22
0xfffffff2aaadb4b0 : 0xffffff7f89ba1d35 com.apple.nvidia.classic.NVDAResmanTesla : _constructObjEHeap + 0x51d3
0xfffffff2aaadb4d0 : 0xffffff7f89abb048 com.apple.nvidia.classic.NVDAResmanTesla : _NvRmDupObject + 0x3b5
0xfffffff2aaadb580 : 0xffffff7f89ab8a84 com.apple.nvidia.classic.NVDAResmanTesla : _insert_registration_func + 0x240
0xfffffff2aaadb770 : 0xffffff7f89ab9b25 com.apple.nvidia.classic.NVDAResmanTesla : _NvRmAlloc + 0x84
0xfffffff2aaadb850 : 0xffffff7f893ff3af com.apple.GeForceTesla : __ZN17nvEvoChannelTesla5FlushEP17nvPushBufferTesla + 0x57
0xfffffff2aaadb890 : 0xffffff7f8940e8f3 com.apple.GeForceTesla : __ZN17nvPushBufferTesla5FlushEv + 0x19
0xfffffff2aaadb8b0 : 0xffffff7f89435498 com.apple.GeForceTesla : __ZN26nvTeslaDisplayManagerTesla35TriggerReleaseOfRenderableSemaphoreEjRj + 0xe4
0xfffffff2aaadb8f0 : 0xffffff7f894354f4 com.apple.GeForceTesla : __ZN26nvTeslaDisplayManagerTesla19EmitChannelBeamSyncEP13__GLNVdataRecjjjjPj + 0x48
0xfffffff2aaadb920 : 0xffffff7f894142e1 com.apple.GeForceTesla : __ZN14nvSurfaceTesla12generateSyncEjjj + 0x15b
0xfffffff2aaadb970 : 0xffffff7f893e1caf com.apple.GeForceTesla : __ZN16IONVSurfaceTesla11submit_swapEj7eDoSwapjP18IONVGLContextTesla + 0x309
0xfffffff2aaadba00 : 0xffffff7f893dde83 com.apple.GeForceTesla : __ZN16IONVSurfaceTesla13flush_surfaceEjj + 0x7f
0xfffffff2aaadba30 : 0xffffff7f893dbbd0 com.apple.GeForceTesla : __ZN16IONVSurfaceTesla13surface_flushEjj + 0x104
0xfffffff2aaadba70 : 0xffffff800867a896 mach_kernel : _shim_io_connect_method_scalarI_scalarO + 0x306
0xfffffff2aaadbb10 : 0xffffff8008678f71 mach_kernel : __ZN12IOUserClient14externalMethodEjP25IOExternalMethodArgumentsP24IOExternalMethodDispatchP8OSObjectPv + 0x271
0xfffffff2aaadbb60 : 0xffffff80086830ec mach_kernel : _is_io_connect_method + 0x42c
0xfffffff2aaadbcc0 : 0xffffff8007f842a6 mach_kernel : _iokit_server_routine + 0x4f56
0xfffffff2aaadbdd0 : 0xffffff8007e569eb mach_kernel : _ipc_kmsg_send + 0x46b
0xfffffff2aaadbe60 : 0xffffff8007e6d2f9 mach_kernel : _mach_msg_overwrite_trap + 0x2c9
0xfffffff2aaadbef0 : 0xffffff8007fb05ba mach_kernel : _mach_call_munger64 + 0x1ea
0xfffffff2aaadbfa0 : 0xffffff8007e1c256 mach_kernel : _hndl_mach_scall64 + 0x16
      Kernel Extensions in backtrace:
         com.apple.driver.AppleMuxControl(6.5.7)[760602C8-782F-3F28-B7AB-524F269F6453]@0xffffff7f88ede000->0xffffff7f88eeffff
            dependency: com.apple.AppleGraphicsDeviceControl(6.5.7)[0BD23394-7F7E-371E-9335-E1FD4E53E857]@0xffffff7f88eb2000->0xffffff7f88eb5fff
            dependency: com.apple.driver.AppleGraphicsControl(6.5.7)[42C4F7D7-1AB9-33B1-81FB-52062FC7638A]@0xffffff7f88e9a000->0xffffff7f88e9afff
            dependency: com.apple.iokit.IOACPIFamily(1.4)[9B7E5F8A-0357-37DF-AC13-BEFEA2714FA3]@0xffffff800a578000->0xffffff800a579fff
            dependency: com.apple.iokit.IOGraphicsFamily(597)[BA342E9C-5821-3A4D-917C-FA32EB96B587]@0xffffff7f8962b000->0xffffff7f89659fff
            dependency: com.apple.iokit.IOPCIFamily(2.9)[49A13CB8-C698-3079-8A27-74A3B097E492]@0xffffff800aa21000->0xffffff800aa4dfff
         com.apple.nvidia.classic.NVDAResmanTesla(10.0.4)[F8258F0F-61AD-30F4-BDD5-B18CCAAE535A]@0xffffff7f89a63000->0xffffff7f89c7bfff
            dependency: com.apple.iokit.IOGraphicsFamily(597)[BA342E9C-5821-3A4D-917C-FA32EB96B587]@0xffffff7f8962b000->0xffffff7f89659fff
            dependency: com.apple.iokit.IONDRVSupport(597)[1D2D1AE7-9A92-3C02-B4E0-4AD99EF97514]@0xffffff7f89674000->0xffffff7f8967cfff
            dependency: com.apple.iokit.IOPCIFamily(2.9)[49A13CB8-C698-3079-8A27-74A3B097E492]@0xffffff800aa21000->0xffffff800aa4dfff
         com.apple.GeForceTesla(10.0.4)[6DB35165-76BB-306C-BE87-E69580EAE417]@0xffffff7f893c1000->0xffffff7f89448fff
            dependency: com.apple.iokit.IOGraphicsFamily(597)[BA342E9C-5821-3A4D-917C-FA32EB96B587]@0xffffff7f8962b000->0xffffff7f89659fff
            dependency: com.apple.iokit.IONDRVSupport(597)[1D2D1AE7-9A92-3C02-B4E0-4AD99EF97514]@0xffffff7f89674000->0xffffff7f8967cfff
            dependency: com.apple.iokit.IOPCIFamily(2.9)[49A13CB8-C698-3079-8A27-74A3B097E492]@0xffffff800aa21000->0xffffff800aa4dfff
            dependency: com.apple.nvidia.classic.NVDAResmanTesla(10.0.4)[F8258F0F-61AD-30F4-BDD5-B18CCAAE535A]@0xffffff7f89a63000->0xffffff7f89c7bfff
         com.apple.nvidia.classic.NVDANV50HalTesla(10.0.4)[5DCF031B-F81F-32A3-ADCD-18E16EAC7CF0]@0xffffff7f897bb000->0xffffff7f89989fff
            dependency: com.apple.iokit.IOPCIFamily(2.9)[49A13CB8-C698-3079-8A27-74A3B097E492]@0xffffff800aa21000->0xffffff800aa4dfff
            dependency: com.apple.nvidia.classic.NVDAResmanTesla(10.0.4)[F8258F0F-61AD-30F4-BDD5-B18CCAAE535A]@0xffffff7f89a63000->0xffffff7f89c7bfff

Process name corresponding to current thread (0xffffff9079489540): WindowServer
Boot args: keepsyms=1 debug=0x100 -lilubetaall -btlfxallowanyaddr ipc_control_port_options=0 -nokcmismatchpanic amfi=0x80

Mac OS version:
21G419

Kernel version:
Darwin Kernel Version 21.6.0: Mon Dec 19 20:44:01 PST 2022; root:xnu-8020.240.18~2/RELEASE_X86_64
Kernel UUID: 33C30416-3302-3FFF-9686-A557DED28C5B
KernelCache slide: 0x0000000007c00000
KernelCache base:  0xffffff8007e00000
Kernel slide:      0x0000000007c10000
Kernel text base:  0xffffff8007e10000
__HIB  text base: 0xffffff8007d00000
System model name: MacBookPro6,2 (Mac-F22586C8)
System shutdown begun: NO
Panic diags file available: YES (0x0)
Hibernation exit count: 0

System uptime in nanoseconds: 683920780036
Last Sleep:           absolute           base_tsc          base_nano
  Uptime  : 0x0000009f3cdaf789
  Sleep   : 0x0000000000000000 0x0000000000000000 0x0000000000000000
  Wake    : 0x0000000000000000 0x000000057966ef9c 0x0000000000000000
Compressor Info: 0% of compressed pages limit (OK) and 0% of segments limit (OK) with 0 swapfiles and OK swap space
Zone info:
  Zone map: 0xffffff80785ec000 - 0xffffffa0785ec000
  . PGZ   : 0xffffff80785ec000 - 0xffffff80795ed000
  . VM    : 0xffffff80795ed000 - 0xffffff8546053000
  . RO    : 0xffffff8546053000 - 0xffffff86df91f000
  . GEN0  : 0xffffff86df91f000 - 0xffffff8bac385000
  . GEN1  : 0xffffff8bac385000 - 0xffffff9078deb000
  . GEN2  : 0xffffff9078deb000 - 0xffffff9545851000
  . GEN3  : 0xffffff9545851000 - 0xffffff9a122b8000
  . DATA  : 0xffffff9a122b8000 - 0xffffffa0785ec000
  Metadata: 0xffffffffdc7ee000 - 0xfffffffffc7ee000
  Bitmaps : 0xfffffffffc7ee000 - 0xffffffffff7ee000

**Comment by queenofcorgis** [comment]
Hi @moranjaccov ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Apple products.  It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by moranjaccov** [reply]
hii , thanks for the reply, im looking for the right topic but couldnt find nothing could u please write the steps to go and open in the roght topic

sorry

**Comment by pabloisco177** [comment]
F
